bciers/libs/shared/components/src/index.ts

```diff
@@ -1,3 +1,4 @@
 // Use this file to export React client components (e.g. those with 'use client' directive) or other non-server utilities
 
-export * from './lib/sample/Sample';
+export * from "./lib/sample/Sample";
+export * from "./lib/theme/ThemeRegistry";
```

Comment:
wouldn't this overwrite default exports?

Reply:
Since I only used named exports, I removed the default export in `Sample` to make it consistent. Changed in https://github.com/bcgov/cas-registration/pull/1264/commits/cdc0c2dfbfc63db7829b1326a1d4249b8a4bf208